apiserver/crossmodel/crossmodel.go

```diff
@@ -218,7 +238,30 @@ func makeOfferParamsFromOffer(url string, offer jujucrossmodel.ApplicationOffer)
 // FindApplicationOffers gets details about remote applications that match given filter.
 func (api *API) FindApplicationOffers(filters params.OfferFilters) (params.FindApplicationOffersResults, error) {
 	var result params.FindApplicationOffersResults
+	offers, err := api.getApplicationOffersDetails(filters)
+	if err != nil {
+		return result, err
+	}
+	for _, offer := range offers {
+		result.Results = append(result.Results, offer.ApplicationOffer)
+	}
+	return result, nil
+}
+
+// ListApplicationOffers gets deployed details about application offers that match given filter.
+// The results contain details about the deployed applications such as connection count.
+func (api *API) ListApplicationOffers(filters params.OfferFilters) (params.ListApplicationOffersResults, error) {
```

Comment:
Normally a split like this would be done so the call that didn't need as much information could avoid the extra cost of gathering it. If the diffrerence is just how the results are displayed why make the split at the API level? I guess splitting the calls now means we can introduce the optimisation later, is that the reasoning?